nixos/modules/programs/firejail.nix

```
@@ -40,13 +40,13 @@ in {
           executable = mkOption {
             type = types.path;
             description = "Executable to run sandboxed";
-            example = literalExample "''${lib.getBin pkgs.firefox}/bin/firefox";
+            example = literalExample ''"''${lib.getBin pkgs.firefox}/bin/firefox"'';
```

This is an escaping nightmare. We need to do something about this.

---

Alas. Nix would have to provide a syntax for non-interpolated literal strings.

---

Perhaps a `"`-string is still easier to read in this case, because `''${` is kind of an awful escape sequence.
```suggestion
            example = literalExample "\"\${lib.getBin pkgs.firefox}/bin/firefox\"";
```

---

I personally think the backslashes look even worse. This is purely personal preference though, I don't think we should enforce either style.

---

Backslashes are at least a different character and not a repetition of the same character 6 times with different spacings